src/main/resources/reference.conf

```diff
@@ -0,0 +1,13 @@
+# librato {
+#   username = "myUsername"
+#   token = "myToken"
+#   source = "mySource"
+# }
+
+cache {
+  windowSize = 6
+  opSize = 2
+  # Note: the location string will have format called on it like: _.format(LayerId.name, LayerId.zoom)
+  # Commented out because not required
+  # location = "/tmp/model-lab/cache/%1$s/%2$s"
```

Comment:
Just wanted to confirm, if this setting if left commented out, the cache is effectively disabled, correct?